src/App.tsx

```diff
@@ -1,61 +1,121 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const [title, setTitle] = useState('');
+  const [hasTitleErr, setHasTitleErr] = useState(false);
+  const [selectUserId, setSelectUserId] = useState(0);
+  const [hasSelectErr, setHasSelectErr] = useState(false);
+
+  const getBiggerId = (arrayItemsWithId: Todo[]) => {
+    const maxId = Math.max(...arrayItemsWithId.map(item => item.id));
+
+    return maxId + 1;
+  };
+
+  const handleTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    const text = value.length === 1 ? value.trim() : value;
```

Comment:
It doesn't fix the problem, I still can add just spaces here. Consider adding validation error if there are just spaces

Reply:
it is working, you maybe checked when change was not downloaded yet